plugins/uefi/fu-plugin-uefi.c

```diff
@@ -91,6 +91,81 @@ fu_plugin_get_results (FuPlugin *plugin, FuDevice *device, GError **error)
 	return TRUE;
 }
 
+static void
+fu_plugin_uefi_add_hsi_tainted (FuPlugin *plugin, GPtrArray *attrs)
+{
+	FwupdSecurityAttr *attr = fwupd_security_attr_new ("org.kernel.CheckTainted");
+	fwupd_security_attr_set_name (attr, "Linux Kernel");
+	fwupd_security_attr_add_flag (attr, FWUPD_SECURITY_ATTR_FLAG_RUNTIME_ISSUE);
+	if (fu_uefi_read_file_as_uint64 ("/proc/sys/kernel", "tainted") == 0) {
```

Comment:
Didn't we grow a `/proc` path lookup as part of swap that should be used?

Reply:
We did indeed, will fix.

Reply:
Done.